include/cadmium/engine/pdevs_dynamic_coordinator.hpp

```diff
@@ -220,17 +235,17 @@ namespace cadmium {
                     //clean outbox because messages are routed before calling this function at a higher level
                     _outbox = cadmium::dynamic::message_bags();
 
-                    LOGGER::template log<cadmium::logger::logger_info>(formatter::log_info_advance, _last, t, _model_id);
+                    LOGGER::template log<cadmium::logger::logger_info, cadmium::logger::coor_info_advance>(_last, t, _model_id);
 
                     if (_next < t || t < _last ) {
                         throw std::domain_error("Trying to obtain output when out of the advance time scope");
                     } else {
 
                         //Route the messages standing in the outboxes to mapped inboxes following ICs and EICs
-                        LOGGER::template log<cadmium::logger::logger_message_routing>(formatter::log_routing_ic_collect, _model_id);
+                        LOGGER::template log<cadmium::logger::logger_message_routing, cadmium::logger::coor_routing_ic_collect>(t, _model_id);
                         cadmium::dynamic::engine::route_internal_coupled_messages_on_subcoordinators<TIME, LOGGER>(_internal_coupligns);
 
-                        LOGGER::template log<cadmium::logger::logger_message_routing>(formatter::log_routing_eic_collect, _model_id);
+                        LOGGER::template log<cadmium::logger::logger_message_routing, cadmium::logger::coor_routing_eic_collect>(t, _model_id);
```

Comment:
I would call the formatter "coordinator_routing_eic", to avoid wondering if coordinator is abreviated coord or coor.

Reply:
Ok, added to my todo list to be made in the next iteration